src/Data/Data.php

```diff
@@ -80,9 +81,20 @@ public static function handler(string $type): Handler
 	/**
 	 * Decodes data with the specified handler
 	 */
-	public static function decode($string, string $type): array
-	{
-		return static::handler($type)->decode($string);
+	public static function decode(
+		$string,
+		string $type,
+		bool $exceptions = true
```

Comment:
`Questions`: 
- What do you think of the parameter name?
- Is this ok for just the `Data` class or would we need to implement it for each handler `::decode()` too?

Reply:
- Another option could be `$fail`. I think we have that somewhere else, but I don't remember where. But `$exceptions` works for me as well.  
- Hm, we could probably use the Data class as a wrapper which would be nice because the handlers don't have to deal with it. Would also probably be a lot more past-proof for other plugin handlers.

Reply:
Renamed it to `$fail`